src/rgw/rgw_data_sync.cc

```diff
@@ -747,12 +764,16 @@ class RGWListBucketIndexesCR : public RGWCoroutine {
   int i;
 
   bool failed;
+  string list_marker;
+  int max_entries;
+  bool truncated;
+  read_metadata_list result;
 
 public:
   RGWListBucketIndexesCR(RGWDataSyncEnv *_sync_env,
-                         rgw_data_sync_status *_sync_status) : RGWCoroutine(_sync_env->cct), sync_env(_sync_env),
+                         rgw_data_sync_status *_sync_status, const string& _list_marker, bool _truncated, read_metadata_list _result) : RGWCoroutine(_sync_env->cct), sync_env(_sync_env),
```

Comment:
shouldn't need to add any extra arguments to the constructor; `truncated(false)` is probably the only new variable that it needs initialize below